examples/tide/actors.rs

```diff
@@ -0,0 +1,155 @@
+use webauthn_rs::ephemeral::WebauthnEphemeralConfig;
```

Comment:
I'm curious if you have a better name for this than Actor, now that it's not an actor at all.

Reply:
The concept is still pretty similar so I'd just leave it for now.